Tests/ArgumentParserUnitTests/HelpGenerationTests.swift

```diff
@@ -592,7 +592,7 @@ extension HelpGenerationTests {
 
   func testHelpWithPrivate() {
     // For now, hidden and private have the same behaviour
```

Comment:
This comment isn't accurate anymore - does this test still make sense?

Reply:
The test still adds coverage but I think I'll replace these visibility tests in a future PR.